.github/dependatbot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+- package-ecosystem: 'npm'
+  directory: '/'
+  schedule:
+    interval: 'daily'
+  open-pull-requests-limit: 10
+  labels:
+  - skip-changelog
+  - dependencies
+  ignore:
+  - dependency-name: 'eslint*'
+  - dependency-name: 'prettier*'
+  rebase-strategy: disabled
+- package-ecosystem: github-actions
+  directory: '/'
+  schedule:
+    interval: daily
+  labels:
+    - skip-changelog
+    - dependencies
```

Comment:
What is this for ?

Reply:
It's for update the github-actions in `test.yml` or GA files :
`actions/checkout@v1` -> `actions/checkout@v2`
`actions/setup-node@v1` -> `actions/setup-node@v2`